index.d.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference types="node" />
```

Comment:
is this needed? I have another typings file where node core libraries are used, but they don't have this line

Reply:
I don't think it is required, but it's documented here https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-

Reply:
ok great